content/zh/docs/setup/production-environment/turnkey/gce.md

```diff
@@ -0,0 +1,397 @@
+2. Examples of how to use the cluster
+3. How to delete the cluster
+4. How to start clusters with non-default options (like larger clusters) 
+-->
+1. 如何在你的工作站设置命令行客户端来管理集群
+2. 如果使用集群的示例
+3. 如何删除集群
+4. 如果以非默认选项启动集群（如规模较大的集群）
+
+<!-- ## Installing the Kubernetes command line tools on your workstation -->
+## 在你的工作站安装 Kubernetes 命令行工具
+
+<!--  
+The cluster startup script will leave you with a running cluster and a `kubernetes` directory on your workstation.
+-->
+集群启动脚本将在你的工作站上留下一个正在运行的集群和一个 `kubernetes` 目录。
+
+<!--  
+The [kubectl](/docs/reference/kubectl/kubectl/) tool controls the Kubernetes cluster
+manager.  It lets you inspect your cluster resources, create, delete, and update
+components, and much more. You will use it to look at your new cluster and bring
+up example apps.
+-->
+[kubectl](/zh/docs/reference/kubectl/kubectl/) 工具控制 Kubernetes 集群管理器。
+它允许你检查集群资源，创建、删除和更新组件等等。
+你将使用它来查看新集群并打开示例应用程序。
+
+<!--  
+You can use `gcloud` to install the `kubectl` command-line tool on your workstation:
+-->
+你可以使用 `gcloud` 在工作站上安装 `kubectl` 命令行工具:
+
+```shell
+gcloud components install kubectl
+```
+
+{{< note >}}
+<!--  
+The kubectl version bundled with `gcloud` may be older than the one
+downloaded by the get.k8s.io install script. See [Installing kubectl](/docs/tasks/tools/install-kubectl/)
+document to see how you can set up the latest `kubectl` on your workstation.
+-->
+与 `gcloud` 绑定的 kubectl 版本可能比 get.k8s.io 下载的更老。
+查看[安装 kubectl](/zh/docs/tasks/tools/install-kubectl/) 文档，了解如何在工作站上设置最新的 `kubectl`。
+{{< /note >}}
+
+<!-- ## Getting started with your cluster -->
+## 开始使用你的集群
+
+<!-- ### Inspect your cluster -->
+### 检查你的集群
+
+<!--  
+Once `kubectl` is in your path, you can use it to look at your cluster. E.g., running:
+-->
+一旦 `kubectl` 存在于你的路径中，你就可以使用它来查看集群，例如，运行：
+
+```shell
+kubectl get --all-namespaces services
+```
+
+<!--  
+should show a set of [services](/docs/concepts/services-networking/service/) that look something like this:
+-->
+应该显示 [services](/zh/docs/concepts/services-networking/service/) 集合，看起来像这样：
+
+```shell
+NAMESPACE     NAME          TYPE             CLUSTER_IP       EXTERNAL_IP       PORT(S)        AGE
+default       kubernetes    ClusterIP        10.0.0.1         <none>            443/TCP        1d
+kube-system   kube-dns      ClusterIP        10.0.0.2         <none>            53/TCP,53/UDP  1d
+kube-system   kube-ui       ClusterIP        10.0.0.3         <none>            80/TCP         1d
+...
+```
+
+<!--  
+Similarly, you can take a look at the set of [pods](/docs/concepts/workloads/pods/) that were created during cluster startup.
+You can do this via the
+-->
+类似的，你可以查看在集群启动时创建的 [pods](/zh/docs/concepts/workloads/pods/) 的集合，
+你可以通过命令：
+
+```shell
+kubectl get --all-namespaces pods
+```
+
+<!--  
+You'll see a list of pods that looks something like this (the name specifics will be different):
+-->
+你将会看到 Pod 的列表，看起来像这样（名称和细节会有所不同）：
+
+```shell
```

Comment:
remove 'shell'

Reply:
done, remove  `shell` for all shell output